packages/deployment/Dockerfile.sdk

```diff
@@ -1,3 +1,5 @@
+# syntax=docker/dockerfile:1.4
```

Comment:
what happens without this?

Reply:
My docker engine won't grok `--link` directives without it.

Reply:
See https://hub.docker.com/r/docker/dockerfile